### DISCUSSION TITLE
Any analog for AMD? Or is this not necessary?

### DISCUSSION THREAD

**Discussion by WhiteKinkyclone** [terminatorul/NvStrapsReBar#45] [General]
Any analog for AMD? Or is this not necessary?

**Comment by xCuri0** [comment]
There's barely anyone still using AMD GPUs before Polaris so not much is known about them. I think @psyborg55 can answer this though.

Everything Polaris and newer supports Resizable BAR fully just that you might have to enable it in registry for driver to use it.

**Comment by psyborg55** [comment]
can't give you a good answer on that one. what i have here is HD3200 (integrated in northbridge, ) and ancient Radeon9200...
i think there is still quite a bit of users on R9 and similar series

**Reply by xCuri0** [reply]
@psyborg55 didn't you figure out AMD also has some straps that control BAR size before ?

**Reply by terminatorul** [reply]
Hi @psyborg55

If you have some documentation on such straps bits for old AMD GPUs please share...

**Reply by psyborg55** [reply]
true, there are hw straps that can set different BAR size, i confirmed this on polaris. but i have no idea about older GPUs as i don't have one. i found the technical docs on some server while repairing my laptop GPU, there were almost all polaris series and one or two vega docs, i did not save vega docs, but somebody else might have them
